python_transport/wirepas_gateway/protocol/mqtt_wrapper.py

```diff
@@ -171,19 +190,21 @@ def _get_socket(self):
         if sock is not None:
             return sock
 
-        self.logger.error("MQTT, unexpected disconnection")
-
-        if not self.connected:
+        if self.connected:
```

Comment:
If it is only needed here, I guess self._client.is_connected() should be enough. No need to duplicate the mqtt connection status here.

Reply:
It seems that is_connected() is available since paho 1.5.0, and we are still using 1.4.0

Reply:
Good point! I probably selected wrong version when checking ;-)